anki-editor.el

```diff
@@ -1021,13 +1029,11 @@ Return a list of cons of (FIELD-NAME . FIELD-CONTENT)."
 
 (defun anki-editor--concat-fields (field-names field-alist level)
   "Concat field names and content of fields in list `field-names'."
-  (let ((format (anki-editor-entry-format)))
```

Comment:
It seems that this code has never been used. @slotThe , please let me know if it breaks anything for you.

Reply:
I've tried this in various ways, and everything seems to work as before :+1: